app/controllers/application_controller.rb

```diff
@@ -20,6 +20,9 @@ def with_current_request
     json_response({:errors => e.message}, :forbidden)
   rescue ManageIQ::API::Common::IdentityError => e
     json_response({:errors => e.message}, :unauthorized)
+  rescue ManageIQ::API::Common::ApiError => e
```

Comment:
@lindgrenj6 Should this become a rescue from in the common api gem.
Maybe we can start adding a mixin for common api gem with all the rescue froms.
@bdunne This might be a use case that will help all of our other services.

Reply:
Oh yeah I like that idea, I'll get to building out that mixin.